es2-persistence/src/main/java/com/netflix/conductor/dao/es/index/ElasticSearchDAO.java

```diff
@@ -182,15 +182,15 @@ private void updateIndexName(Configuration config) {
 	 */
 	private void initIndex() throws Exception {
 
-		//0. Add the index template
-		GetIndexTemplatesResponse result = elasticSearchClient.admin().indices().prepareGetTemplates("wfe_template").execute().actionGet();
+		//0. Add the tasklog template
+		GetIndexTemplatesResponse result = elasticSearchClient.admin().indices().prepareGetTemplates("tasklog_template").execute().actionGet();
 		if (result.getIndexTemplates().isEmpty()) {
-			logger.info("Creating the index template 'wfe_template'");
```

Comment:
Just to confirm, is this no longer needed?

Reply:
Yes, tasklog_template would suffice the templates required for conductor. Others are defined in docType mappings.